setup.py

```diff
@@ -25,6 +24,19 @@ def fetch_requirements(path):
         return [r.strip() for r in fd.readlines()]
 
 
+def available_vector_instructions():
+    cpu_vector_instructions = {}
+    try:
+        import cpufeature
+        cpu_vector_instructions = cpufeature.CPUFeature
+    except _:
```

Comment:
Could we first `except ImportError` to catch the case of `cpufeature` not being installed? It will only be available if users manually install our requirements first.

Reply:
Done!